src/lib/exports.f90

```diff
@@ -438,11 +444,11 @@ end subroutine export_data_geometry
   subroutine export_terminal_solution(EXNODEFILE, name)
   !DEC$ ATTRIBUTES DLLEXPORT,ALIAS:"SO_EXPORT_TERMINAL_SOLUTION" :: EXPORT_TERMINAL_SOLUTION
 
-    use arrays,only: elem_nodes,&
-         node_xyz,num_units,units,unit_field
-    use indices,only: nu_comp,nu_pe,nu_vt,nu_vent
-    use other_consts, only: MAX_FILENAME_LEN, MAX_STRING_LEN
-    implicit none
+!    use arrays,only: elem_nodes,&
```

Comment:
delete - i am going to stop writing these now - but best to delete all these please and avoid leaving commented out lines.